.changelog/289.trivial.md

```diff
@@ -0,0 +1 @@
+Discover: Backport discovery fixes to stable branch
```

Comment:
@csillag , probably best to classify this as "Feature" so that Towncrier will not delete all the Change Log fragments when assembling the Change Log for 1.1.3?